spec/support/shared_examples/signable_configuration.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
this file is used in 1 place.. why a shared_example?

Reply:
When we wrote this we thought we'd be using it in multiple test files. I've moved it to `client_spec` now.